mmv1/templates/terraform/examples/artifact_registry_repository_cleanup.tf.erb

```diff
@@ -0,0 +1,33 @@
+resource "google_artifact_registry_repository" "<%= ctx[:primary_resource_id] %>" {
+  location      = "us-central1"
```

Comment:
once min_version is added `provider= google-beta` will be needed on all resources within the test
```suggestion
  provider = google-beta
  location      = "us-central1"
```

Reply:
I had this written up but forgot to commit!. Thanks.